scripts/tests/test_commit_gitops_manifests.py

```
@@ -0,0 +1,170 @@
+    auth_env = _git_auth_env(inputs.gitops_token, tmp_path)
+    assert commit_and_push(inputs, clone_dir, auth_env) is None
+
+
+def test_commit_and_push_dry_run(monkeypatch: pytest.MonkeyPatch, tmp_path: Path) -> None:
+    inputs = _make_inputs(tmp_path, dry_run=True)
+    clone_dir = tmp_path / "clone"
+    clone_dir.mkdir()
+
+    calls: list[list[str]] = []
+
+    def fake_run_git(args: list[str], _cwd: Path, env: dict[str, str] | None = None) -> str:
```

Ruff (0.14.13)

Unused function argument: `env`

(ARG001)